### PR TITLE
Ports Enforcer handgun, rubber bullets, and new shotgun ammo boxes from Stratus

### PR DESCRIPTION
**Pull request by Allfd**
This PR ports the Enforcer handgun as an admin only NT version of the syndicate 10mm pistol.

The Enforcer is capable of firing lethal and rubber rounds, 9mm rubber rounds have also been added.

Also ported are a variety of new shotgun ammo boxes.

Image of Enforcer PR

🆑 Stratus
rscadd: NT Enforcer .45 pistol.
rscadd: .45 and 9mm rubber bullets.
imageadd: New shotgun ammo boxes.
/🆑

### CODE REVIEW

**Comment by Spacemanspark** [comment]
What's Stratus

**Comment by Allfd** [comment]
@Spacemanspark Its a thing I was told to credit

**Comment by Spacemanspark** [comment]
Do they sell memes

**Comment by Allfd** [comment]
I have no idea.

**Comment by Spacemanspark** [comment]
We must find out immediately.
@FreeStylaLT

**Comment by FlattestGuitar** [comment]
:eyes:

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 1]
```diff
@@ -40,6 +40,14 @@
 	mag_type = /obj/item/ammo_box/magazine/m50
 	can_suppress = 0
 
+/obj/item/weapon/gun/projectile/automatic/pistol/enforcer45
+	name = "Enforcer .45"
+	desc = "A pistol of modern design. Standard issue for Nanotrasen Asset Protection units."
```
Comment (FlattestGuitar) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Something else as per lore, perhaps?

I don't think there is "Asset Protection" on Paradise.

**Comment by FlattestGuitar** [comment]
If you port the shotty sprites from here it will be dope.

**Comment by Fox-McCloud** [comment]
@FlattestGuitar Nu

**Comment by Spacemanspark** [comment]
Why not, they actually look pretty damn badass.

File: code/modules/projectiles/guns/projectile/pistol.dm (hunk 1 above)
Comment (SomanBat) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Paradise has Shitcurity.

Stratus has AssPros.

Lets keep our lore correct kiddos.

Comment (Allfd) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Nobody on Cyberiad is getting these, can change of course.

NanoTrasen Asset Protection is in the lore.

Comment (KingPhilipIII) [inline code/modules/projectiles/guns/projectile/pistol.dm]: Isn't "Asset Protection" another name for DeathSquad on Para?

Comment (Allfd) [inline code/modules/projectiles/guns/projectile/pistol.dm]: @KingPhilipIII It is.

**Comment by Fox-McCloud** [comment]
Is this supposed to be a Glock?

**Comment by Spacemanspark** [comment]
Yes.

**Comment by Allfd** [comment]
It looks more like a H&K Mk23,

Glocks don't have an external safety.

**Comment by DesolateG** [comment]
Fox us a Kimber fan

**Comment by Fox-McCloud** [comment]
awww

Muh Glock

**Comment by FlattestGuitar** [comment]
I just sprited it to look cool. Glock was an obvious inspiration here, but the thing is just whatever.

**Comment by FalseIncarnate** [comment]
I'd recommend changing the last changelog entry ("rscadd: New shotgun ammo boxes") to be "imageadd: New shotgun ammo box sprites", as this did not actually add new types of shotgun ammo boxes, just new sprites for them

**Comment by Allfd** [comment]
@FalseIncarnate done

**Pull request merged**